cmd/meroxa/global/metrics_test.go

```diff
@@ -93,7 +93,7 @@ func TestAddError(t *testing.T) {
 	}
 
 	err := "unexpected error"
-	addError(event, fmt.Errorf("%s", err))
+	addError(event, err)
```

Comment:
oh wait sorry I was blind:
```
err := errors.New("unexpected error""
addError(event, err)
```

Reply:
Got it, thank you!

Reply:
Perfect, this works great and thank you for the outline!